skimage/restoration/_denoise.py

```diff
@@ -3,10 +3,11 @@
 from .. import img_as_float
 from ..restoration._denoise_cy import _denoise_bilateral, _denoise_tv_bregman
 from .._shared.utils import _mode_deprecations
+import warnings
 
 
 def denoise_bilateral(image, win_size=5, sigma_range=None, sigma_spatial=1,
-                      bins=10000, mode='constant', cval=0):
+                      bins=10000, mode='constant', cval=0, multichannel=False):
     """Denoise image using bilateral filter.
```

Comment:
I think the default value of ``multichannel`` should be ``True``, since the case of 2-D images is more frequent than 3-D images.